dockerfiles/devicehive-frontend-rdbms/devicehive-start.sh

```diff
@@ -79,6 +79,7 @@ java -server -Xms1g -Xmx2g -XX:+UseG1GC -XX:MaxGCPauseMillis=20 -XX:+DisableExpl
 -Drpc.client.response-consumer.threads="${DH_RPC_CLIENT_RES_CONS_THREADS:-3}" \
 -Dserver.context-path=/api \
 -Dserver.port=8080 \
+-Dauth.base.url="${DH_AUTH_URL}" \
```

Comment:
Frontend launch script should check this parameter before starting application, something like
```
  || [ -z "$DH_AUTH_URL" ] \
```
in the beginning of the script

Reply:
Done!